qiskit/scheduler/__init__.py

```diff
@@ -19,24 +19,17 @@
 
 .. currentmodule:: qiskit.scheduler
 
-A scheduler compiles a circuit program to a pulse program.
+A circuit scheduler compiles a circuit program to a pulse program.
 
 .. autosummary::
    :toctree: ../stubs/
 
    schedule_circuit
    ScheduleConfig
 
-Scheduling utility functions
-
-.. autosummary::
-   :toctree: ../stubs/
-
-   qiskit.scheduler.utils
-
 .. automodule:: qiskit.scheduler.methods
 """
 
 from qiskit.scheduler.config import ScheduleConfig
-from qiskit.scheduler.schedule_circuit import schedule_circuit
+from qiskit.scheduler import schedule_circuit
```

Comment:
This requires deprecation; for instance, a user who had:
```
from qiskit.scheduler import schedule_circuit

schedule_circuit(...)
```
would get an error

Reply:
This is an internal method and not user-facing in the API?

Reply:
Why do you say it's internal, just because it's unlikely someone is using it?

I'll leave this to @ajavadia

Reply:
bump